dlt/extract/source.py

```diff
@@ -333,6 +334,11 @@ def add_limit(self, max_items: int) -> "DltSource":  # noqa: A003
             resource.add_limit(max_items)
         return self
 
+    def parallelize(self) -> "DltSource":
+        for resource in self.resources.selected.values():
```

Comment:
cool but IMO we should skip resources that cannot be parallelized. so those that will raise exception from resource.parallelize. also docstrings